cli/template/page-studs/index/with-auth-trpc-tw.tsx

```diff
@@ -64,10 +64,13 @@ const Home: NextPage = () => {
 export default Home;
 
 const AuthShowcase: React.FC = () => {
-  const { data: secretMessage } = trpc.auth.getSecretMessage.useQuery();
-
   const { data: sessionData } = useSession();
 
+  const { data: secretMessage } = trpc.auth.getSecretMessage.useQuery(
+    undefined,
```

Comment:
Can be even more verbose while we're at it

```suggestion
    undefined, // no input
```